front/lib/api/assistant/conversation.ts

```diff
@@ -242,6 +244,9 @@ async function renderUserMessage(
       email: userMessage.userContextEmail,
       profilePictureUrl: userMessage.userContextProfilePictureUrl,
     },
+    reactions: message.reactions
+      ? renderMessageReactions(message.reactions)
```

Comment:
I don't see where message.reactions is set? Am I missing something?

Reply:
It's loaded directly on the message object, see change on `getConversation` line 407!

Reply:
I see. I really feel like this is not enough business critical to be loaded always at all time. wdyt?